homeassistant/components/http/ban.py

```diff
@@ -51,12 +51,11 @@ def setup_bans(hass: HomeAssistant, app: Application, login_threshold: int) -> N
     app.middlewares.append(ban_middleware)
     app[KEY_FAILED_LOGIN_ATTEMPTS] = defaultdict(int)
     app[KEY_LOGIN_THRESHOLD] = login_threshold
+    app[KEY_BAN_MANAGER] = IpBanManager(hass)
```

Comment:
I'd probably write this as `ban_manager = app[KEY_BAN_MANAGER] = IpBanManager(hass)` and than await it below